src/ImageHorizonLibrary/recognition/_recognize_images.py

```diff
@@ -203,9 +208,11 @@ def try_locate(ref_image):
         center_point = ag.center(location)
         x = center_point.x
         y = center_point.y
-        if self.has_retina:
-            x = x / 2
-            y = y / 2
+        if self.pixel_ratio == 0.0:
+            self.__get_pixel_ratio()
+        if self.pixel_ratio>1:
```

Comment:
> If this were changed to if not isclose(self.pixel_ratio, 1.0, abs_tol=1e-3):, would this allow it to work for large (>1) or small (<1) scaling ratios?

Actually I could simply do:
```
if self.pixel_ratio>0 and self.pixel_ratio != 1.0:
```
 To cover the same case, then there would be no need to import isclose.

I have no idea how we could test it though? Windows only allows scaling from 100% to 500% so would never have a scaling ratio below 1 and MacOS only had 100% for non retina displays and 200% for retina displays no other scaling ratios, I'm not aware of any linux distro that supports scaling below 100% either.

I could actually just use too
```
if self.pixel_ratio>0:
```
as any number divided by 1 will give the same number, but it seems a waste of processing power to do divide by 1 twice, but it might make the code easier to read?

let me know the preferred option and i'll update the PR.

Dave.